OWNERS.md

```diff
@@ -79,6 +80,10 @@ contributors to envoy-setec and relevant Slack channels from:
   * Alessandro Gario ([alessandrogario](https://github.com/alessandrogario))
   * Mike Myers ([mike-myers-tob](https://github.com/mike-myers-tob))
 
+* [Feb 2022 Security Release Contributors]
```

Comment:
I think we're just adding them both to EST no?

Reply:
/wait

Reply:
Cool.  LGTM but can you get a thumbs up from a non-googler on setec for this one?